packages/manager/modules/carrier-sip/src/endpoints/translations/Messages_fr_FR.json

```
@@ -0,0 +1,12 @@
+{
+  "carrier_sip_endpoints": "SIP Endpoints",
+  "carrier_sip_endpoints_id": "ID",
```

```suggestion
  "carrier_sip_endpoints_id": "Id",
```
This is the short for `identifier`, not an acryonym, even if Content says it is 😄